src/x86/sse41.rs

```diff
@@ -724,6 +788,87 @@ mod tests {
     }
 
     #[simd_test = "sse4.1"]
+    unsafe fn _mm_packus_epi32() {
+        let a = i32x4::new(1, 2, 3, 4);
+        let b = i32x4::new(-1, -2, -3, -4);
+        let r = sse41::_mm_packus_epi32(a, b);
+        let e = u16x8::new(1, 2, 3, 4, 0, 0, 0, 0);
+        assert_eq!(r, e);
+    }
+
+    #[simd_test = "sse4.1"]
+    unsafe fn _mm_cmpeq_epi64() {
+        let a = i64x2::new(0, 1);
+        let b = i64x2::new(0, 0);
+        let r = sse41::_mm_cmpeq_epi64(a, b);
+        let e = i64x2::new(-1, 0);
+        assert_eq!(r, e);
+    }
+
+     #[simd_test = "sse4.1"]
+    unsafe fn _mm_cvtepi8_epi16() {
+        let a = i8x16::splat(10);
+        let r = sse41::_mm_cvtepi8_epi16(a);
+        let e = i16x8::splat(10);
+        assert_eq!(r, e);
+        let a = i8x16::splat(-10);
+        let r = sse41::_mm_cvtepi8_epi16(a);
+        let e = i16x8::splat(-10);
+        assert_eq!(r, e);
+    }
+
+    #[simd_test = "sse4.1"]
+    unsafe fn _mm_cvtepi8_epi32() {
+        let a = i8x16::splat(10);
+        let r = sse41::_mm_cvtepi8_epi32(a);
+        let e = i32x4::splat(10);
+        assert_eq!(r, e);
+        let a = i8x16::splat(-10);
+        let r = sse41::_mm_cvtepi8_epi32(a);
+        let e = i32x4::splat(-10);
+        assert_eq!(r, e);
+    }
+
+    /*
+    #[simd_test = "sse4.1"]
+    unsafe fn _mm_cvtepi8_epi64() {
+        let a = i8x16::splat(10);
+        let r = sse41::_mm_cvtepi8_epi64(a);
+        let e = i64x2::splat(10);
+        assert_eq!(r, e);
+        let a = i8x16::splat(-10);
+        let r = sse41::_mm_cvtepi8_epi64(a);
+        let e = i64x2::splat(-10);
+        assert_eq!(r, e);
+    }
+    */
+
+    #[simd_test = "sse4.1"]
+    unsafe fn _mm_cvtepi16_epi32() {
+        let a = i16x8::splat(10);
+        let r = sse41::_mm_cvtepi16_epi32(a);
+        let e = i32x4::splat(10);
+        assert_eq!(r, e);
+        let a =  i16x8::splat(-10);
+        let r = sse41::_mm_cvtepi16_epi32(a);
+        let e = i32x4::splat(-10);
+        assert_eq!(r, e);
+    }
+
+/*
+    #[simd_test = "sse4.1"]
+    unsafe fn _mm_cvtepi16_epi64() {
+        let a = i16x8::splat(10);
+        let r = sse41::_mm_cvtepi16_epi64(a);
+        let e = i64x2::splat(10);
+        assert_eq!(r, e);
+        let a =  i16x8::splat(-10);
+        let r = sse41::_mm_cvtepi16_epi64(a);
+        let e = i64x2::splat(-10);
+        assert_eq!(r, e);
+    }
+*/
```

Comment:
Why is this test commented out?

Reply:
It was temporarily commented until the respective intrinsic could be implemented